yelp_package/dockerfiles/lucid/Dockerfile

```diff
@@ -20,6 +20,9 @@ MAINTAINER Kyle Anderson <kwa@yelp.com>
 # Which at this time is in this package
 RUN apt-get update && apt-get -y install dpkg-dev python-tox python-setuptools python-dev libssl-dev libffi-dev debhelper dh-virtualenv python-yaml libyaml-dev python-pytest pyflakes python2.7 python2.7-dev help2man zsh git
 
+RUN pip install -U pip
+RUN pip install -U virtualenv
```

Comment:
This may fix the build temporarily, but this kind of pip install is what causes breakage in the long term because it make all of our hard work of pinning stuff go to waste